ios/noiseScore/AppDelegate.m

```diff
@@ -31,7 +31,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 {
 
   NSURL *jsCodeLocation;
-  [GMSServices provideAPIKey:@"{GOOGLE_CLOUD_API_KEY}"]; // add this line using the api key obtained from Google Console
+  [GMSServices provideAPIKey:@"AIzaSyAfGO8M5ocLOiSV8CmPE-TRh8ifEXbtGqo"]; // add this line using the api key obtained from Google Console
```

Comment:
Do we want this API key to be publicly exposed in the repo?

Reply:
Yeah I forgot to update that. But it's fixed now